### PR TITLE
fix(github-ids): update all github id's data types to BigInt in schema

### PR DESCRIPTION
**Pull request by rockingrohit9639**
# Summary

Updated all github related id's datatype to `BigInt`

## Related Issues

Which issue does this PR resolve?

#158

### CODE REVIEW

File: prisma/migrations/20240307174047_update_github_id_datatype_to_bigint/migration.sql [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ALTER TABLE "users" ALTER COLUMN "githubUserId" SET DATA TYPE BIGINT;
+
+-- AddForeignKey
+ALTER TABLE "synced_issues" ADD CONSTRAINT "synced_issues_githubRepoId_fkey" FOREIGN KEY ("githubRepoId") REFERENCES "github_repos"("repoId") ON DELETE RESTRICT ON UPDATE CASCADE;
+
+-- AddForeignKey
+ALTER TABLE "syncs" ADD CONSTRAINT "syncs_githubRepoId_fkey" FOREIGN KEY ("githubRepoId") REFERENCES "github_repos"("repoId") ON DELETE RESTRICT ON UPDATE CASCADE;
+
+-- AddForeignKey
+ALTER TABLE "milestones" ADD CONSTRAINT "milestones_githubRepoId_fkey" FOREIGN KEY ("githubRepoId") REFERENCES "github_repos"("repoId") ON DELETE RESTRICT ON UPDATE CASCADE;
```
Comment (tedspare) [inline prisma/migrations/20240307174047_update_github_id_datatype_to_bigint/migration.sql:35]: How was this migration generated?

**Review by tedspare — CHANGES_REQUESTED** [review]
One comment otherwise LGTM

File: prisma/migrations/20240307174047_update_github_id_datatype_to_bigint/migration.sql (hunk 1 above)
Comment (rockingrohit9639) [reply prisma/migrations/20240307174047_update_github_id_datatype_to_bigint/migration.sql:35]: I ran prisma migration command `pnpm prisma migrate dev`

**Review by tedspare — APPROVED** [review]
LGTM

**Pull request merged**